mlir/test/Dialect/Tosa/invalid.mlir

```diff
@@ -64,11 +106,83 @@ func.func @test_pad_non_const(%arg0: tensor<13x21x3xf32>, %arg1: tensor<3x2xi32>
 
 // -----
 
-func.func @test_pad_non_const(%arg0: tensor<13x21x3xi8>, %arg1: tensor<i8>) -> tensor<13x21x3xi8> {
+func.func @test_pad_non_const(%arg0: tensor<13x21x3xi8>, %arg1: tensor<i8>) -> tensor<?x?x?xi8> {
```

Comment:
Why did we need to change the test case?

Reply:
The output shape in the test case is incorrect. It was not throwing an error earlier. With my changes to fix this, we are now seeing the output mismatch error instead of the expected "pad_const is not a constant" error.